be/src/common/config.h

```diff
@@ -835,9 +835,6 @@ CONF_Int32(starlet_fs_stream_buffer_size_bytes, "131072");
 
 CONF_mInt64(lake_metadata_cache_limit, /*2GB=*/"2147483648");
 CONF_mBool(lake_print_delete_log, "true");
-CONF_mInt64(lake_gc_metadata_max_versions, "10");
-CONF_mInt64(lake_gc_metadata_check_interval, /*30 minutes=*/"1800");
-CONF_mInt64(lake_gc_segment_check_interval, /*60 minutes=*/"3600");
```

Comment:
some other configs are not used, such as lake_enable_aggressive_gc, lake_aggressive_gc_high_priority.